modules/pipeline/providers/cron/compensator/provider.go

```diff
@@ -57,6 +58,7 @@ type provider struct {
 
 	jsonStore jsonstore.JsonStore
 	dbClient  *dbclient.Client
+	client    *db.Client
```

Comment:
rename to `cronDBClient` or just `cronDB`. `client` is not clear, client for what :(

Reply:
remove `op_pipelien_cron` inside legacy `dbclient` pkg.